src/shared/components/common/progress-bar.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from "classnames";
+import { ThemeColor } from "shared/utils";
+
+type ProgressBarProps = {
+  className?: string;
+  backgroundColor?: ThemeColor;
+  barColor?: ThemeColor;
+  striped?: boolean;
+  animated?: boolean;
+  min?: number;
+  max?: number;
+  value: number;
+  text?: string;
+};
+
+const ProgressBar = ({
```

Comment:
I went with the lambda style function over the normal style here because it's just directly returning a result. I can understand why some prefer the normal style function over the lambda style when the function needs to do multiple statements, but this seems like the kind of situation lambda functions were made for.